session/manager.go

```diff
@@ -221,13 +224,23 @@ func (manager *Manager) Acknowledge(consumerID identity.Identity, sessionID stri
 	return nil
 }
 
+func (manager *Manager) clearStaleSession(consumerID identity.Identity, serviceType string) {
+	sessionID, ok := manager.sessionStorage.FindBy(FindOpts{
```

Comment:
BTW, should not it be a list of customer stale sessions we should find in storage?

Reply:
If you are creating second session, first one will be destroyed. It's impossible to have more than 1 session.